Source/radiation/Radiation.cpp

```diff
@@ -138,6 +138,11 @@ void Radiation::read_static_params()
     do_multigroup = 0;
   }
 
+  // check that we're not using a single group solver if NGROUPS > 1
+  if (!do_multigroup && NGROUPS > 1) {
```

Comment:
If we've filled in Radiation::nGroups by now (I think we have?) could you check on that instead?